### PR TITLE
Website/ Connect Mailchimp to NewsletterSignup

### PR DESCRIPTION
**Pull request by DavidBachmann**
# Mailchimp connection

https://app.asana.com/0/1183498341031105/1195333141378473

## What

Make a form component for the website that uses island-ui's NewsletterSignup and sends signups to Mailchimp.

## Why

So we can get people on our mailing list

## Screenshots / Gifs

Screenshot 2020-09-23 at 14 18 38
Screenshot 2020-09-23 at 14 18 44
Screenshot 2020-09-23 at 14 21 13

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

File: libs/island-ui/core/src/lib/NewsletterSignup/NewsletterSignup.tsx [hunk 1]
```diff
@@ -18,12 +17,13 @@ interface Props {
   label: string
   buttonText: string
   variant?: ColorVariant
-  onClickSubmit: () => void
   state?: State
   errorMessage?: string
+  onChange: (bla: any) => void
```
Comment (siggakogk) [inline libs/island-ui/core/src/lib/NewsletterSignup/NewsletterSignup.tsx:22]: maybe change 'bla' to something more descriptive?

Comment (DavidBachmann) [reply libs/island-ui/core/src/lib/NewsletterSignup/NewsletterSignup.tsx:22]: :'D

Comment (DavidBachmann) [reply libs/island-ui/core/src/lib/NewsletterSignup/NewsletterSignup.tsx]: Fixed in https://github.com/island-is/island.is/pull/1018/commits/743a7b5951c17dce9d8badd302088d8d209a7037

**Review by siggakogk — APPROVED** [review]
Looks good!

**Pull request merged**